pfp/index.html

```diff
@@ -147,21 +227,140 @@
                 /* margin-left: 20px; */
             }
 
+            /* Adding elements with bootstrap d-print-none class added, so external stylesheet for html2pdf works */
```

Comment:
This note was originally for when I was using an external stylesheet, so that specific wording no longer applies. But the rule still works.